docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -1785,10 +1893,10 @@ from_unixtime(expression)
 - [array_sort](#array_sort)
 - [array_cat](#array_cat)
 - [array_concat](#array_concat)
-- [array_contains](#array_contains)
```

Comment:
this is not related to PR?

Reply:
Indeed you are correct, there are likely a couple of places where I fixed things as I saw them. Would you like those pulled into a separate PR?

Reply:
I removed this from the PR